agimus_controller_ros/agimus_controller_ros/agimus_controller_parameters.yaml

```diff
@@ -52,3 +52,15 @@ agimus_controller_params:
     type: double
     default_value: 100.0
     description: "Rate of mpc node in hertz"
+  free_flyer:
```

Comment:
I am wondering if we should get that from the lfc as well. And maybe the srdf file too...
Maybe for another PR?

Reply:
yeah, would be nice. Indeed, let's leave for the next PR. We can discuss tomorrow maybe